### PR TITLE
lbreakout2: init at 2.6.5

### PR DESCRIPTION
**Pull request by ciil**
###### Things done
Just added a run of the mill default expression, and also myself to maintainers, as this is my first PR to nixpkgs.

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/lbreakout2/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, SDL, SDL_mixer, zlib, libpng, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "lbreakout2-${version}";
+  version = "2.6.5";
+  buildInputs = [ SDL SDL_mixer zlib libpng gcc ];
```
Comment (joachifm) [inline pkgs/games/lbreakout2/default.nix]: gcc is already added to the build environment via stdenv (unless cc is overridden). Is there a particular reason for adding it explicitly?

File: pkgs/games/lbreakout2/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/lgames/${name}.tar.gz";
```
Comment (joachifm) [inline pkgs/games/lbreakout2/default.nix]: Please use `mirror://sourceforge`

**Review by joachifm — CHANGES_REQUESTED** [review]
Thank you. I've added a few notes for you to consider.

**Comment by joachifm** [comment]
Testing contributions with sandboxing enabled is preferred, as it helps reduce the risk of implicit dependencies.

**Review by joachifm — APPROVED** [review]

**Comment by ciil** [comment]
Thanks! Will definitely test with sandboxing next time.

**Comment by joachifm** [comment]
The travis error is legit.

**Comment by ciil** [comment]
Yeah, there's no license.gpl. Also the license is actually GPLv2, which I didn't notice, because I can't read *smh*. Should be good now, also built sandboxed.

**Comment by joachifm** [comment]
LGTM. Thank you

**Pull request merged**